index.html

```diff
@@ -32,6 +32,34 @@
 
     <link rel="stylesheet" media="all" href="styles/styles.min.css" />
   </head>
+  <body>
+    <input id="toggle-menu" class="menu__element--input" type="checkbox" />
+    <div class="body__apla"></div>
+    <nav class="menu">
+      <div class="menu__container">
+        <div class="menu__col--left">
+          <div class="menu__logo">
+            <a href="#"><img src="images/logo.svg" alt="Easybank logo" /></a>
+          </div>
+    <style>
+      .attribution {
```

Comment:
@avtill coś tu poszło nie tak, duplikaty są.
Dodatkowo jak buduję u siebie brancha WC-I-2, to wygląda tak:
![screencapture-127-0-0-1-5500-index-html-2023-05-21-21_58_18](https://github.com/sabiku85/easybank-lp/assets/22479664/8e505f7b-b339-4f20-b17e-bbd9720abea8)